geode-memcached/src/main/java/org/apache/geode/internal/memcached/MemcachedSanctionedSerializablesService.java

```diff
@@ -12,20 +12,16 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.codeAnalysis;
+package org.apache.geode.internal.memcached;
 
+import java.net.URL;
 
-import org.apache.geode.gfsh.internal.management.GfshDistributedSystemService;
+import org.apache.geode.internal.serialization.SanctionedSerializablesService;
 
-public class AnalyzeGfshSerializablesJUnitTest extends AnalyzeSerializablesJUnitTestBase {
+public class MemcachedSanctionedSerializablesService implements SanctionedSerializablesService {
```

Comment:
Note: github diffs are confused... the code is correct.

AnalyzeGfshSerializablesJUnitTest was renamed to AnalyzeGfshSerializablesIntegrationTest which still exists in geode-gfsh.

MemcachedDistributedSystemService was renamed to MemcachedSanctionedSerializablesService which still exists in geode-memcached.